corehq/motech/repeaters/static/repeaters/js/bootstrap3/repeat_record_report.js

```diff
@@ -205,16 +205,15 @@ hqDefine('repeaters/js/bootstrap3/repeat_record_report', [
             return $.find('input[type=checkbox][name=record_ids]:checked');
         }
 
-        function isActionPossibleForCheckedItems(action, items) {
-            for (const item of items) {
-                const id = item.getAttribute('data-id');
-                const query = `[data-record-id="${id}"][class="btn btn-default ${action}-record-payload"]`;
-                const button = document.querySelector(query);
-                if (!button) {
+        function isActionPossibleForCheckedItems(action, checkedItems) {
+            for (const item of checkedItems) {
+                const isQueued = item.getAttribute('is_queued');
+                if (isQueued == 'false' && action == 'cancel') {
```

Comment:
Why is `isQueued` checking for false? It looks like the values are `0`/`1`, or possibly `"0"`/`"1"` since `getAttribute` returns a string.

Reply:
The suggestion below is the best way I can think of, although I don't love that it takes three steps, from string to int to boolean.

You can set a boolean in HTML, but you have to omit it instead of saying `my_val="false"`, which I think is less explicit/readable (I'm looking at [these docs](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML)).

Reply:
Ok cool. Ended up going with your suggestion below in ea7eaf4463e40497b32e46e13256d2048bc4c4cc